src/components/StaticImage.test.tsx

```diff
@@ -14,9 +14,7 @@ describe('<StaticImage/>', () => {
     });
 
     it('prepends the asset prefix to the img src', () => {
-      expect(__PATH_PREFIX__).toBe('/docs');
-
-      const { getByTestId } = render(<StaticImage src="/images/example.png" />);
+      const { getByTestId } = render(<StaticImage src={'/docs/images/example.png'} />);
```

Comment:
@jamiehenson I think these should not actually have changed and they are hiding an issue that we see as broken images on the homepage.

The `src` on the homepage for LiveSync is `/docs/static/1740413963000127948506/bf86f/livesync.png` but on disk the asset is at `public/static/1740413963000127948506/bf86f/livesync.png`, so no `/docs` in the path.

When we deploy to production the value of `ASSET_PREFIX` kicks in and the `src` will be `https://docs.ably.com/static/1..../livesync.png`, solving the routing issue through the website.